<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1318</width>
    <height>839</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Principal</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
background-color: rgb(255, 255, 255);
 }
QToolTip {
	color: #ffffff;
	background-color: rgba(27, 29, 35, 160);
	border: 1px solid rgb(40, 40, 40);
	border-radius: 2px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QStackedWidget" name="swMain">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:  rgb(255, 255, 255);</string>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="main">
         <layout class="QGridLayout" name="gridLayout_8"/>
         <widget class="QFrame" name="frame">
          <property name="geometry">
           <rect>
            <x>186</x>
            <y>224</y>
            <width>371</width>
            <height>288</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>371</width>
            <height>288</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>371</width>
            <height>288</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>2</width>
            <height>2</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">padding: 3px;
border-color: rgb(255, 85, 0);
border-top-color: rgb(255, 0, 0);
</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>182</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="btnResearcher">
             <property name="minimumSize">
              <size>
               <width>220</width>
               <height>71</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Roboto</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
background-color: rgb(94, 184, 255);
border: none;
color: rgb(255, 255, 255);
gridline-color:transparent;
}
QPushButton:hover {
	background-color: rgb(0, 91, 159);
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}</string>
             </property>
             <property name="text">
              <string>Estou Pesquisando</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QFrame" name="frame_2">
          <property name="geometry">
           <rect>
            <x>740</x>
            <y>224</y>
            <width>371</width>
            <height>288</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>2</horstretch>
            <verstretch>2</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>371</width>
            <height>288</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>800</width>
            <height>600</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">padding: 3px</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>182</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="btnParticipant">
             <property name="minimumSize">
              <size>
               <width>220</width>
               <height>71</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Roboto</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
background-color: rgb(94, 184, 255);
border: none;
color: rgb(255, 255, 255);
gridline-color:transparent;
}
QPushButton:hover {
	background-color: rgb(0, 91, 159);
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}</string>
             </property>
             <property name="text">
              <string>Sou Participante</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="researcher">
         <layout class="QGridLayout" name="gridLayout_10"/>
         <widget class="QFrame" name="frame_4">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>60</y>
            <width>991</width>
            <height>512</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">
</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="3" column="2">
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>229</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="4" column="1">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="1" rowspan="2">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QLabel" name="lError">
                 <property name="font">
                  <font>
                   <family>Roboto</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(255, 0, 0);</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="leEmail">
                 <property name="minimumSize">
                  <size>
                   <width>491</width>
                   <height>81</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>491</width>
                   <height>81</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Roboto</family>
                   <pointsize>16</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(245, 245, 246);
border: None;
border-bottom: 1px solid blue;
color: rgb(116, 116, 116);</string>
                 </property>
                 <property name="text">
                  <string>joelbisponeto@gmail.com</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="placeholderText">
                  <string>E-mail</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lePassword">
                 <property name="minimumSize">
                  <size>
                   <width>491</width>
                   <height>81</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>491</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Roboto</family>
                   <pointsize>16</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(245, 245, 246);
border: None;
border-bottom: 1px solid blue;
color: rgb(116, 116, 116);</string>
                 </property>
                 <property name="text">
                  <string>123456</string>
                 </property>
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="placeholderText">
                  <string>Senha</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnLogin">
                 <property name="minimumSize">
                  <size>
                   <width>491</width>
                   <height>71</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>491</width>
                   <height>71</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Roboto</family>
                   <pointsize>20</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
background-color: rgb(94, 184, 255);
border: none;
color: rgb(255, 255, 255);
gridline-color:transparent;
}
QPushButton:hover {
	background-color: rgb(0, 91, 159);
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}</string>
                 </property>
                 <property name="text">
                  <string>ENTRAR</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QPushButton" name="btnForgetPassword">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>201</width>
                   <height>51</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Roboto</family>
                   <pointsize>12</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
background-color:transparent;
border: none;
color:  rgb(2, 136, 209);
gridline-color:transparent;
}
QPushButton:hover {
background-color:transparent;
border-bottom: 1px solid blue;
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}</string>
                 </property>
                 <property name="text">
                  <string>Esqueci minha senha</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnRegister">
                 <property name="minimumSize">
                  <size>
                   <width>201</width>
                   <height>51</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>201</width>
                   <height>51</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Roboto</family>
                   <pointsize>12</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
background-color:transparent;
border: none;
color:  rgb(2, 136, 209);
gridline-color:transparent;
}
QPushButton:hover {
background-color:transparent;
border-bottom: 1px solid blue;
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}</string>
                 </property>
                 <property name="text">
                  <string>Quero me cadastrar</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="0" column="0" colspan="3">
            <widget class="QFrame" name="frame_5">
             <property name="minimumSize">
              <size>
               <width>971</width>
               <height>80</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>971</width>
               <height>80</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(94, 146, 243);</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <widget class="QLabel" name="label_6">
              <property name="geometry">
               <rect>
                <x>310</x>
                <y>20</y>
                <width>481</width>
                <height>41</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <family>Roboto Black</family>
                <pointsize>28</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Área de Pesquisa</string>
              </property>
             </widget>
            </widget>
           </item>
           <item row="2" column="0">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>229</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1">
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="topFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(2, 136, 209);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>768</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QFrame" name="frame_3">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>35</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI Black</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QLabel" name="lUser">
              <property name="font">
               <font>
                <family>Roboto</family>
                <pointsize>16</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
